voucher/src/main/res/layout/voucher_view.xml

```diff
@@ -22,7 +22,7 @@
 
     <TextView
         android:id="@+id/textView_description"
-        style="@style/AdyenCheckout.Voucher.DescriptionTextAppearance"
+        style="@style/AdyenCheckout.Voucher.Introduction.Bacs"
```

Comment:
Why exactly are we renaming it? And why is introduction better than description?

Reply:
I just wanted to make it like Boleto for consistency